velox/exec/tests/TaskTest.cpp

```diff
@@ -697,4 +697,60 @@ DEBUG_ONLY_TEST_F(TaskTest, outputDriverFinishEarly) {
   // Wait for Values driver to complete.
   driverFuture.wait();
 }
+
+// To avoid "redefinition of 'TaskTest_DISABLED_test_name_Test'" errors.
+class TaskTestExtra : public TaskTest {};
+
+/// Test that we export operator stats for unfinished (running) operators.
+DEBUG_ONLY_TEST_F(TaskTestExtra, liveStats) {
```

Comment:
We can't define more than one test cases using DEBUG_ONLY_TEST_F?

Reply:
It seems so. I've got error both in sandcastle and in GH. :(

Reply:
I think it is a bug in
```
#define DEBUG_ONLY_TEST(test_fixture, test_name) \
  TEST(test_fixture, DISABLED_##test_name)
#define DEBUG_ONLY_TEST_F(test_fixture, test_name) \
  TEST_F(test_fixture, DISABLED_test_name)
#define DEBUG_ONLY_TEST_P(test_fixture, test_name) \
  TEST_P(test_fixture, DISABLED_test_name)
```
Can you help change to
```
#define DEBUG_ONLY_TEST(test_fixture, test_name) \
  TEST(test_fixture, DISABLED_##test_name)
#define DEBUG_ONLY_TEST_F(test_fixture, test_name) \
  TEST_F(test_fixture, DISABLED_##test_name)
#define DEBUG_ONLY_TEST_P(test_fixture, test_name) \
  TEST_P(test_fixture, DISABLED_##test_name)
```